SU-HSDO/suhumsci
# fix(shs-5105): gin form override for input width overlap

# READY FOR REVIEW

## Summary
This PR adds a small override to the Gin theme (coming from the humsci_basic theme) to constrain form text input fields to 100%. 

## Urgency
medium

## Steps to Test
1. Edit the homepage in both hs_colorful and hs_traditional
2. Edit either the "Testimonial" or "Spotlight" component 
3. Bring your browser down to a tablet sized width (before it switches to a single column)
4. Note that the fields are no longer breaking out of their parent container. 

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
